les/handler.go

```diff
@@ -868,6 +868,9 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 						}
 						root = header.Root
 					}
+					if root == (common.Hash{}){
```

Comment:
Could you add a comment here (just so it's clear when someone maintains this later on) that:

`// If a header lookup failed (non existent), ignore subsequent requests for the same header`

Reply:
Also pls gofmt